pkgs/top-level/python-packages.nix

```diff
@@ -10776,6 +10774,8 @@ in modules // {
     };
   };
 
+  gdal = pkgs.gdal.override { pythonPackages = self; };
```

Comment:
gdal is not a python package, but it is shipped with python bindings. The passthru is just enough for the current use.

I added it here to have the gdal module available for each python package set (if someone wants to do a `nix-shell -p python35Packages.gdal`).

The current state is:

```bash
$ nix-shell -p gdal -p pythonPackages.python --command "python -c 'import gdal'"
$ nix-shell -p gdal -p python35Packages.python --command "python3 -c 'import gdal'"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named 'gdal'
```

It will probably be simpler for the current PR to just use the passthru. There is no urgent need for the various gdal derivations, and I am not really satisfied with that since it duplicates all the gdal derivation just to have different bindings. The rest of the derivation us just the same (bins, libs…). I’ll update the PR asap.